src/api/fixtures/AutolendProfile.js

```diff
@@ -0,0 +1,37 @@
+import _omit from 'lodash/omit';
+import LoanSearchCriteria, { criteriaAreEqual, getInputCriteria } from './LoanSearchCriteria';
+
+// Return an AutolendProfile object with default values
+export default function AutolendProfile() {
+	return {
+		__typename: 'AutolendProfile',
+		id: null,
+		isEnabled: false,
+		donationPercentage: 15,
+		enableAfter: 0,
+		loanSearchCriteria: LoanSearchCriteria(),
+	};
+}
+
+// Return a cleaned profile suitable for input
+export function getInputProfile(profile) {
+	const cleanProfile = _omit(profile, [
```

Comment:
The lodash docs call out omit as being much slower than pick so in this case it may be better to pick 3 instead of omit 3... but we can wait and see if it's an issue first.